packages/mrmime/mrmime.0.1.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+name:         "mrmime"
+version:      "dev"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/mirage/mrmime"
+bug-reports:  "https://github.com/mirage/mrmime/issues"
+dev-repo:     "git+https://github.com/mirage/mrmime.git"
+doc:          "https://mirage.github.io/mrmime/"
+license:      "MIT"
+synopsis:     "Mr. MIME"
+description:  """Parser and generator of mail in OCaml"""
```

Comment:
I would more see this in synopsis instead and have either an even more elaborate description or just remove the description field.